core-primitives/settings/src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub mod files {
 /// Settings concerning the worker
 pub mod worker {
 	// the maximum size of any extrinsic that the enclave will ever generate in B
-	pub const EXTRINSIC_MAX_SIZE: usize = 4196;
+	pub const EXTRINSIC_MAX_SIZE: usize = 5500;
```

Comment:
It may be only used there for now, but it is of general importance if one wants to generate a new extrinsic via Ecall. As this an SDK I believe we can leave this parameter here. Or are you strongly against that?

Reply:
Ok, let's leave it like this